src/Services/Adapters/SafetyPayRequestAdapter.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace Ebanx\Benjamin\Services\Adapters;
+
+class SafetyPayRequestAdapter extends RequestAdapter
```

Comment:
Yeah, I know it shouldn't have added it now. But I have plans for it. SafetyPay is the only gateway that needs a `-` in its payment type code. Safety Pay Online will also need this Adapter.